<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextBrowser" name="msgBrowser">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>491</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="msgTextEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>481</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="usrTableWidget">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>201</width>
     <height>411</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>用户名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IP</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>440</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>在线人数:0人</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFontComboBox" name="fontCbx"/>
    </item>
    <item>
     <widget class="QComboBox" name="sizeCbx">
      <property name="currentText">
       <string>16</string>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="boldBtn">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/image/B1.png</normaloff>:/image/B1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="italicBtn">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/image/B2.png</normaloff>:/image/B2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="colorBtn">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/image/B3.png</normaloff>:/image/B3.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="underlineBtn">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/image/B4.png</normaloff>:/image/B4.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="sendBtn_2">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/image/B5.png</normaloff>:/image/B5.png</iconset>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="saveBtn">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/image/B6.png</normaloff>:/image/B6.png</iconset>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="clearBtn">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/image/B7.png</normaloff>:/image/B7.png</iconset>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
